base/bin/migration.mustache

```diff
@@ -2,9 +2,9 @@ const fs = require('fs');
 const mustache = require('mustache');
 
 const now = new Date().toISOString();
-const migrationName = now + '_' + process.argv[2];
+const migrationName = `${now}_${process.argv[2]}`;
```

Comment:
It's probably fine, but we should update the test scripts and CI and stuff to be using node 6 if that's what we're targeting.